nvim-completion.test.nvim fails

```
+run_vim_test:2> nvim --headless -c 'source /home/runner/work/dotfiles/dotfiles/.dotfiles/nvim/test-support.vim' -c 'source /home/runner/work/dotfiles/dotfiles/.dotfiles/nvim/lsp-completion.test.vim' -c 'call RunTest()'
[69945](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69946)
Running on test12.py
[69946](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69947)
lsp_init: 0
[69947](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69948)
Starting to waittest_wait: 0
[69948](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69949)
Error:  Error detected while processing function RunTest[2]..Test:
[69949](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69950)
line    7:
[69950](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69951)
Test failed for test12.py
[69951](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69952)
Running on test12.py
[69952](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69953)
lsp_init: 0
[69953](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69954)
Starting to wait
[69954](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69955)
test_wait: 0
[69955](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69956)
Error:  Error detected while processing function RunTest[2]..Test:
[69956](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69957)
line    7:
[69957](https://github.com/kaihowl/dotfiles/runs/7940004318?check_suite_focus=true#step:4:69958)
Error: Process completed with exit code 1.
````

Fails to wait for completion menu to show. When running interactively, the menu shows up. But with https://github.com/hrsh7th/nvim-cmp/commit/db261b4e2b923e4ba5f748b2d20f2a0d066a4ca8 the function `complete_info['pum_visible']` no longer works. This might be because the _native_ window for nvim-lsp is experimental (or maybe deprecated?). 

Double check:
- Is our configuration still adequate?
- Does the user-facing behavior still work as expecte?
- Does the test's waiting function need an update?

--- a/nvim/lsp-completion.test.vim
+++ b/nvim/lsp-completion.test.vim
@@ -1,7 +1,6 @@
 " Test completion with omni-complete and LSP
 function Check(id)
   " Wait until the completion menu is visible.
-  redraw
   echomsg 'Waiting for completion to show'
   let complete_visible = wait(10000, "complete_info()['pum_visible']")
   echomsg 'complete_visible: ' . complete_visible
@@ -16,17 +15,11 @@ function Check(id)
   call feedkeys("\<esc>")
 endfunction
 
-
-function Trigger(id)
-  call timer_start(500, funcref('Check'))
-  call feedkeys("\<c-x>\<c-o>")
-endfunction
-
 function FeedIt(complete_chars)
-  call timer_start(500, funcref('Trigger'))
+  call timer_start(500, funcref('Check'))
   echomsg 'Feeding keys'
   " Get completion for 'inline' keyword
-  call feedkeys('i'.a:complete_chars, 'tx!')
+  call feedkeys('i'.a:complete_chars."\<c-x>\<c-o>", 'tx!')
 endfunction
 
 function ProtoTest(filename, complete_chars)

--- a/nvim/test.sh
+++ b/nvim/test.sh
@@ -32,8 +32,7 @@ function run_vim_test {
 echo "Check that plugins are installed"
 run_vim_test completion.test.vim
 run_vim_test editorconfig.test.vim
-# Disabled #427
-# run_vim_test lsp-completion.test.vim
+run_vim_test lsp-completion.test.vim
 run_vim_test lsp-efm.test.vim
 run_vim_test nvim-cmp-select-enter.test.vim
 run_vim_test restorecurpos.test.vim

